modeling/gbert_classifier.py

```diff
@@ -184,38 +179,79 @@ def eval_model(model, data_loader, loss_fn, device, n_examples):
                 input_ids=input_ids,
                 attention_mask=attention_mask
                 )
-            preds = torch.round(outputs)
+            preds = torch.sigmoid(outputs)
```

Comment:
why did you move the sigmoid into the `eval_model`?

Reply:
Question has been answered in the team call meanwhile. :heavy_check_mark:

For reference: Because of using the different loss function `BCEWithLogitsLoss`